src/schema/v2/artist/helpers.ts

```diff
@@ -190,3 +195,21 @@ export const getAuctionRecord = async (artist, auctionLotsLoader) => {
 
   return highAuctionRecord
 }
+
+const getRecentShow = (artist): string[] => {
+  // dd/mm/yyy|slug|Group or Solo|Show Title
+  const entities = splitEntities(artist.recent_show)
```

Comment:
Can you leave an example artist that actually has this field as non-null? I can't find one, including artists that have current shows in prod.

We're sure this actually conforms to that weird string format? IMO this should have a runtime-codec or something to verify and cast the type.

Reply:
(Ah, OK this is coming from the `artist_insights_importer`)